### PR TITLE
Development

### PR DESCRIPTION
**Pull request by cornets28**
Pull request!

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
Hi, good start for this project. Few changes need to be made

File: index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        
+        <style>
+            @import url('https://fonts.googleapis.com/css?family=Roboto:100');
+            </style>
+    </head>
+    <body> 
+        <div class="wrapp-all">
+            <header class="elements first-item">       
+                <ul>
```
Comment (Oghenebrume50) [inline index.html]: Use a nav tag here instead of a ul tag,  you could even remove the header tag and just keep the nav

File: index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                    <li><img src="images/logo.png"></li>
+                    <li><span>Store</span></li>
+                    <li><span>Mac</span></li>
+                    <li><span>iPod</span></li>
+                    <li><span>iPhone</span></li>
+                    <li><span>iTunes</span></li>
+                    <li><span>iPad</span></li>
+                    <li><span>Support</span></li>
+                    <li><input type="search" name="search" placeholder="&#128269"></li>
```
Comment (Oghenebrume50) [inline index.html]: close the character reference with a semi-colon

File: index.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                </h1>
+               
+                <h2>
+                   Everyone has something to share. See how peopleare using iPad<br>
+                   in unique ways to add to the world's story.
+                 </h2>
+               
+                <h3>
+                   Watch the "Your Verse" film and explore the stories
+                </h3>
```
Comment (Oghenebrume50) [inline index.html]: The content in the main tag are actually images not text follow this bullet point
- [ ] Each link is an <a> HTML tag, with a <div> tag inside it.
- [ ] The <div> tag  inside the <a> tag should use an image as background.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                    
+            </main>
+              
+
+        </div>
+       <br>   <br>   <br>   <br>   <br><br><br><br><br><br><br>
```
Comment (Oghenebrume50) [inline index.html]: remove these `<br>`  and actually use CSS to put in space you need

File: index.html [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        <aside>
```
Comment (Oghenebrume50) [inline index.html]: Move this aside tag and it's content into the footer, follow this guidelines for the footer
## Footer
* [ ] It should be wrapped by `<footer>` tags
* [ ] The top part is made of four `<a>` tags with `<div>` tags inside it
* [ ] The `<div>`s inside the `<a>` tags should use an image as a background
* [ ] The bottom part should include links like copyright, terms of use, and privacy policy

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Make all these changes and submit for re-review. Kudo, for a great start

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Nicely done  👍
Great job on this project. But before merging to master, we need to do some changes to make it in line with Microverse guidelines.

- Navbar

- [ ]  The input tag from the navbar should have the focus effect enabled.

- Main content

- [ ] Main content links (The 3 main sentences)

- [ ] There should be three links (each one inside an anchor tag).

- [ ] Each link is an a tag, with a div tag inside it.  

- [ ] The div tag inside the anchor tag should use an image as background. The text on the original page is actually an image. This approach is required by Microverse guidelines. There is no need to use img tag here. Only a tag and div.

- Footer

- [ ]   The top part is made of four a tags with div tags inside it

- [ ] The divs inside the anchor tags should use an image as a background. The same approach of the 3 main sentences.

Kindly make these changes and submit for code review again!

**Comment by cornets28** [comment]
Hello Sergio,

I successfully found your email. Kindly tell me where I find the
requirements for the project as do not see the for some reasons.
Best regards
Samuel

**Comment by Torres-ssf** [comment]
At the current moment, these guidelines are only available for the TSE team in order to make easy for us to review projects. They were created by the Microverse team. The suggested changes were copied from the guidelines.

**Comment by mosaaleb** [comment]
## Currently reviewing

File: index.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        
+        <style>
+            @import url('https://fonts.googleapis.com/css?family=Roboto:100');
+        </style>
+    </head>
+    <body> 
+        <div class="wrapp-all">
+          <nav>
+              <header class="elements first-item">
```
Comment (mosaaleb) [inline index.html]: There is no `wrapp-all` styles in your css files, its `wrap-all`, may be a misspel. I have tried to apply your styles to this div, but it breaks the page.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+          </nav>
+
+            <main>   
+                <div class="link1">
+                    <a href=""><h1>What will your verce be?</h1></a><br />
+                </div>  
+
+                <div class="link2">
+                    <a href=""><h2>Everyone has something to share. See how people are using <br />
+                      iPad in their unique ways to add to the world's story.</h2></a>
```
Comment (mosaaleb) [inline index.html]: The three text are actually images
- [ ] The `<div>` tag  inside the `<a>` tag should use an image as background -- [ ] The text on the original page is actually an image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+                
+                <div class="link3">
+                    <a href="playVideo"><h3>Watch the "Your Verce" film and explore the stories</h3></a> 
+                </div>
+                                                                                                   
+            </main>
+
+        <footer>
+            <div class="pictures" style="display: flex">
+              <div class="links l2">
```
Comment (mosaaleb) [inline index.html]: It's against best practices to use inline styling as it is harder to debug and also harder to change in the future.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Hi @CornetS28 :wave:

### Well done working on this project :clap:. I know that you have put a lot of effort into this project, yet there is some requirements mentioned by previous reviewers are not met yet. 

See my comments below.

Also I see there is a duplicate styles in both `index.css` and `styles.css` like `wrap-all` class. A way to make sure we don't have duplicates and having healthy style guidelines is to use [`stickler`](https://github.com/microverseinc/linters-config/tree/master/css). I highly recommend to use it before moving on.

Please sumbit a new code review once you handle these requirements.

Feel free to contact me on slack if you have any questions.
## Happy coding :computer:

**Comment by samgaco** [comment]
### Hi @CornetS28 :hand: 

### Good job and congratulations :tada: the project is approved.

Just consider these **suggestions** before merging to the master branch:

* Correct errors in the index file pointed out by https://validator.w3.org/
* Add a short description of the project and what you tried to achieve in the readme file
* Removing the red-colored borders from the navigation bar

### Anyways, the project is approved and you can merge to the master branch and submit the project completion form :muscle: Regards!

**Comment by cornets28** [comment]
Thank you very much!!!

**Pull request merged**